config/redirects

```diff
@@ -1,8 +1,8 @@
 define: base https://docs.mongodb.com
-define: versions v2.2 v2.4 v2.6 v3.0 v3.2 v3.4 v3.6 v4.0 v4.2 v4.4 master
+define: versions v2.2 v2.4 v2.6 v3.0 v3.2 v3.4 v3.6 v4.0 v4.2 v4.4 v5.0 master
 symlink: manual -> v4.4
-symlink: upcoming -> v4.4	
-symlink: master -> v4.4
+symlink: upcoming -> v5.0
```

Comment:
These lines don't do anything in Snooty, but it doesn't hurt to set them either! LGTM :D